test/testdrive/kafka-exactly-once-sink.td

```diff
@@ -0,0 +1,163 @@
+
+> CREATE SOURCE input_csv
+  FROM FILE '${testdrive.temp-dir}/static.csv'
+  FORMAT CSV WITH 3 COLUMNS
+
+> CREATE SINK output1 FROM input_kafka_byo
+  INTO KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'output-view-${testdrive.seed}'
+  WITH (exactly_once=true, consistency=true)
+  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
+
+! CREATE SINK output2 FROM input_kafka_no_byo
+  INTO KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'output-view-${testdrive.seed}'
+  WITH (exactly_once=true, consistency=true)
+  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
+all input sources of a consistent Kafka sink must be replayable, materialize.public.input_kafka_no_byo is not
+
+! CREATE SINK output3 FROM input_table
+  INTO KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'output-view-${testdrive.seed}'
+  WITH (exactly_once=true, consistency=true)
+  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
+all inputs of a consistent Kafka sink must be sources, materialize.public.input_table is not
+
+> CREATE SINK output4 FROM input_kafka_byo_mview
+  INTO KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'output-view-${testdrive.seed}'
+  WITH (exactly_once=true, consistency=true)
+  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
+
+> CREATE SINK output4_view FROM input_kafka_byo_mview_view
+  INTO KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'output-view-${testdrive.seed}'
+  WITH (exactly_once=true, consistency=true)
+  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
+
+! CREATE SINK output5 FROM input_kafka_no_byo_mview
+  INTO KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'output-view-${testdrive.seed}'
+  WITH (exactly_once=true, consistency=true)
+  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
+all input sources of a consistent Kafka sink must be replayable, materialize.public.input_kafka_no_byo is not
+
+> CREATE SINK output5_view FROM input_kafka_no_byo_mview_view
+  INTO KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'output-view-${testdrive.seed}'
+  WITH (exactly_once=true, consistency=true)
+  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
+
+! CREATE SINK output6 FROM input_table_mview
+  INTO KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'output-view-${testdrive.seed}'
+  WITH (exactly_once=true, consistency=true)
+  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
+all inputs of a consistent Kafka sink must be sources, materialize.public.input_table is not
+
+! CREATE SINK output7 FROM input_values_view
+  INTO KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'output-view-${testdrive.seed}'
+  WITH (exactly_once=true, consistency=true)
+  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
+all inputs of a consistent Kafka sink must be sources, materialize.public.input_values_view is not
```

Comment:
This should have succeeded I think. Views created as ```CREATE VIEW ... AS VALUES (...),(...)``` can not be modified after the fact.